src/gtk/webview_webkit2.cpp

```diff
@@ -569,38 +539,35 @@ bool wxWebViewWebKit::Create(wxWindow *parent,
     }
 
     SetupWebExtensionServer();
-    g_signal_connect_after(webkit_web_context_get_default(),
-                           "initialize-web-extensions",
-                           G_CALLBACK(wxgtk_initialize_web_extensions),
-                           m_dbusServer);
+    g_signal_connect(webkit_web_context_get_default(),
```

Comment:
Just for my education: why were all these `connect_after` changed to simple `connect`s? Were they simply wrong before and we should have never let the default behaviour take place before our callsbacks were invoked?

Reply:
The one for `decide-policy` was definitely wrong, as the default signal handler was getting called before us, and our handler never got called.  As best as I could tell, none of the other ones were definitely breaking anything, but I decided to change them all just in case.